poetry.lock

```diff
@@ -162,7 +162,7 @@ tests_no_zope = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>
 
 [[package]]
 name = "azure-core"
-version = "1.18.0"
+version = "1.19.0"
```

Comment:
there shouldn't be a change in this file as you haven't changed / updated any dependencies. Can you please revert this file?